### DISCUSSION TITLE
Lutheran Confessions

### DISCUSSION THREAD

**Discussion by remysheppard** [imfing/hextra#85] [Show and tell]
I built [LutheranConfessions.Org](https://www.lutheranconfessions.org) with Hextra!

Right now it is just the primary confessional document for Lutheran Christians, the _Book of Concord_, but I hope to expand it to include sermons, writings, hymns, art, and much more!

Thanks for creating a great theme!

**Comment by imfing** [comment]
Thank you for sharing it! 
Appreciate your contribution as well. 

Feel free to let me know any issue or feature requests.
